src/screens/login/HSwitchLoginFlow/HyperSwitchAuthUtils.res

```diff
@@ -457,3 +457,9 @@ let errorSubCodeMapper = (subCode: string) => {
   | _ => UR_00
   }
 }
+
+let generateBodyForEmailRedirection = (url: RescriptReactRouter.url) => {
+  let tokenFromUrl =
+    url.search->LogicUtils.getDictFromUrlSearchParams->Dict.get("token")->Option.getOr("")
+  [("token", tokenFromUrl->JSON.Encode.string)]->LogicUtils.getJsonFromArrayOfJson
```

Comment:
Can be done in single line. 

We are not using that value and easy code readability.

```Suggestion
[("token",url.search->LogicUtils.getDictFromUrlSearchParams->Dict.get("token")->Option.getOr("")->JSON.Encode.string)]->LogicUtils.getJsonFromArrayOfJson

Reply:
not needed any separate variable until you are using that somewhere

Reply:
Sure. OK